src/containers/TemplateBuilder/template/General/MessagesConfig.tsx

```diff
@@ -48,7 +48,7 @@ const MessagesConfig: React.FC<MessagesConfigProps> = ({ isOpen, onClose }) => {
   const updateMessage = async () => {
     const { startMessage, submissionMessage } = state
     const result = await updateTemplate(templateId, {
-      startMessage,
+      startMessage: startMessage && startMessage !== '' ? startMessage : null,
```

Comment:
```suggestion
      startMessage: startMessage ? startMessage : null,
```
I think this would work, as empty string evaluates to false.